### DISCUSSION TITLE
不会搞Acess Token...

### DISCUSSION THREAD

**Discussion by ikaoda** [Ivanlon30000/PlayniteBangumiMetadata#3] [Q&A]
用Bangumi账号登录Bangumi API那个网址显示email does not exists or email and password not match
是要怎么申请API呢，小白一个，大佬能帮下么？

**Comment by ikaoda** [comment]
哦，貌似要先去开发者申请，麻

**Comment by Ivanlon30000** [comment]
> @ikaoda 
> 哦，貌似要先去开发者申请，麻

申请AccessToken（个人令牌）不需要成为开发者。

获取AccessToken流程：
1. [Bangumi API 个人令牌 管理页面](https://next.bgm.tv/demo/access-token)
2. 登录Bangumi账号
3. 创建个人令牌 - 输入令牌名称(任意名称) - 选择令牌有效期 - 提交(Submit)

**请保管好你的个人令牌（AccessKey）**，拥有个人令牌相当于拥有Bangumi账户的完全访问权  
如果令牌泄露务必及时在[Bangumi API 个人令牌](https://next.bgm.tv/demo/access-token)删除个人令牌  

PS：部分条目新注册用户看不到，需要注册满一定时间才可见，见https://github.com/Ivanlon30000/PlayniteBangumiMetadata/issues/2#issuecomment-1987608048
